conda/recipes/librmm/recipe.yaml

```
@@ -106,6 +106,7 @@ outputs:
         - ${{ stdlib("c") }}  # this is here to help with overlinking errors against libm.so.6 and friends
       host:
         - ${{ pin_subpackage("librmm", exact=True) }}
+        - cuda-driver-dev
```

Please add this to the top-level `cache.requirements.host` as well.
```suggestion
        - if: cuda_major != "11":
          then: cuda-driver-dev
```